apps/studio/data/logs/unified-logs-count-query.ts

```diff
@@ -0,0 +1,102 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+
+import { getLogsCountQuery } from 'components/interfaces/UnifiedLogs/UnifiedLogs.queries'
+import { FacetMetadataSchema } from 'components/interfaces/UnifiedLogs/UnifiedLogs.schema'
+import { get, handleError } from 'data/fetchers'
+import { ExecuteSqlError } from 'data/sql/execute-sql-query'
+import { logsKeys } from './keys'
+import {
+  getUnifiedLogsISOStartEnd,
+  UNIFIED_LOGS_STALE_TIME,
+  UnifiedLogsVariables,
+} from './unified-logs-infinite-query'
+
+export async function getUnifiedLogsCount(
+  { projectRef, search }: UnifiedLogsVariables,
+  signal?: AbortSignal
+) {
+  if (typeof projectRef === 'undefined') {
+    throw new Error('projectRef is required for getUnifiedLogsCount')
+  }
+
+  const sql = getLogsCountQuery(search)
+  const { isoTimestampStart, isoTimestampEnd } = getUnifiedLogsISOStartEnd(search)
+
+  const { data, error } = await get(`/platform/projects/{ref}/analytics/endpoints/logs.all`, {
+    params: {
+      path: { ref: projectRef },
+      query: {
+        iso_timestamp_start: isoTimestampStart,
+        iso_timestamp_end: isoTimestampEnd,
+        project: projectRef,
+        sql,
+      },
+    },
+    // body: { sql },
```

Comment:
Checking this isn't needed?

Reply:
correct - for context sorry, we're gonna shift the logs.all requests to a POST endpoint for unified logs
just that its dependent on this API PR: https://github.com/supabase/infrastructure/pull/23013